src/System.Console/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uapaot-Windows_NT;
```

Comment:
Why do we need a uap/uapaot configuration? This just segregates the Unix tests into another file, just like should have been done already and we do elsewhere already.

The configuration would be in the case we need #if uap or include files only when compiling for uap which is not the case here.